pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -409,6 +409,26 @@ const (
 	queueImmediately
 )
 
+// isInterestedEvent returns true if the event is interested by some plugins.
+func (p *PriorityQueue) isInterestedEvent(logger klog.Logger, event framework.ClusterEvent) bool {
+	if event.IsWildCard() {
+		return true
+	}
+
+	for _, hintMap := range p.queueingHintMap {
+		for eventToMatch := range hintMap {
+			if eventToMatch.Match(event) {
```

Comment:
Rebased upon the latest master. 
I only changed here to use a new `Match` function introduced in @kerthcet 's PR.
https://github.com/kubernetes/kubernetes/pull/123117